Java/shared/src/main/java/com/lantanagroup/link/shared/logging/DelayedHttpAppender.java

```diff
@@ -0,0 +1,37 @@
+package com.lantanagroup.link.shared.logging;
+
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import com.github.loki4j.logback.Loki4jAppender;
+import org.springframework.boot.context.event.ApplicationReadyEvent;
+import org.springframework.context.ApplicationListener;
+import org.springframework.stereotype.Component;
+
+import java.util.concurrent.ConcurrentLinkedQueue;
+@Component
```

Comment:
Please add a java-doc style comment on this class indicating what its purpose is.